lua/tool_balance/util.lua

```diff
@@ -1,8 +1,13 @@
+local function count(tbl)
```

Comment:
this is for counting tables like this `{1, 2, nil, 4, nil, 6}`  table.Count would 4 when 6 is needed